packages/@aws-cdk/aws-ec2/lib/vpc-ref.ts

```diff
@@ -180,14 +191,14 @@ export abstract class VpcNetworkBase extends Construct implements IVpcNetwork {
   public abstract readonly availabilityZones: string[];
 
   /**
-   * Parts of the VPC that constitute full construction
+   * Dependencies for internet connectivity
    */
-  public readonly dependencyElements: IDependable[] = [];
```

Comment:
Just expose the internet gateway?

Reply:
It's actually not the gateway, it's the attachment of the gateway to the VPC.